docs/RC/OnInteriorVehicleData/index.md

```diff
@@ -0,0 +1,61 @@
+## OnInteriorVehicleData
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Inform SDL about any changes in RC module settings.
+
+!!! must
+
+The HMI must send RC.OnInteriorVehicleData notification to SDL when module settings were changed in case of button press event or after SetInteriorVehicledata request was processed
+
+!!!
+
+### Notification
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|moduleData|[Common.ModuleData](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/Common/Structs/index.md)|||
+
+### Sequence Diagrams
+||| OnInteriorVehicledata
```

Comment:
Fix formatting for chart. Chart not displaying correctly.  See here https://smartdevicelink.com/en/docs/pull_request/f3f90b16d50ad6dbce7a00bbc59f9978495ebd0e/rc/oninteriorvehicledata/

Reply:
fixed